package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "dentoro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development nodemon server/index.js",
+    "kill": "fuser -k 5000/tcp",
+    "server": "npm run dev",
```

Comment:
```suggestion
```

Reply:
Did you want to remove dev and kill scripts?